archlinux/README.md

```diff
@@ -0,0 +1,37 @@
+# Archlinux ARM root builder
+
+## Dependencies
+
+read: https://wiki.archlinux.org/index.php/Raspberry_Pi#QEMU_chroot
+
+```
+yaourt -S binfmt-support qemu-user-static
+systemctl enable binfmt-support
+systemctl start binfmt-support
```

Comment:
you could use `systemctl enable --now binfmt-support` instead of the 2 seperate commands